server/controllers/request.js

```diff
@@ -0,0 +1,85 @@
+
+   
+const Requset = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /request/
+// @desc Get this user all request 
+// @access Private
+exports.getAllRequest = asyncHandler(async (req, res, next) => {
+    const userId  = req.user.id
+    const requests = await Request.find({userId:userId})
```

Comment:
Please be mindful of code style, proper whitespace so we can keep things more readable.

Reply:
Fix throughout the entire PR.

Reply:
This problem is fixed in other PR. After that PR merge, I will rebase this PR .